### PR TITLE
home work react1 week2

### PR DESCRIPTION
**Pull request by melo4yxa77796**
1.created component planetCard
2.created component SocialMediaItem.js
3.created NavItem component
4.moved PlanetWishListItem to its own file
5.added a link to navigate to each of the pages
6.added the functionality to reflect the add or remove of planets and …
Screenshot 2024-10-15 at 20 53 34
Screenshot 2024-10-15 at 20 54 05
Screenshot 2024-10-15 at 20 54 25

### CODE REVIEW

File: app/destination/page.js [hunk 1]
```diff
@@ -1,40 +1,59 @@
...
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
-    </div>
-  );
-}
-
 
 export const Destinations = () => {
   const [selectedPlanets, onAddPlanet] = useState([]);
```
Comment (vdaguenet) [inline app/destination/page.js]: You could initialise the state with your array of planets

File: components/ui/Footer.js [hunk 2]
```diff
@@ -18,47 +27,67 @@ export const Footer = () => {
...
-
-      {/* TASK - React 1 week 1 */}
-      {/* Add a new list item for LINKEDIN */}
-      <div className={styles.footerLinks}>
-        <h3>Follow us</h3>
-        <ul className={styles.footerList}>
+        <ul className={styles.pagesList}>
           <li>
-            <a href="https://facebook.com">Facebook</a>
+            {" "}
```
Comment (vdaguenet) [inline components/ui/Footer.js]: You don't need these forced empty spaces
```suggestion
            
```

File: components/ui/Navbar.js [hunk 3]
```diff
@@ -35,25 +49,20 @@ export const Navbar = () => {
...
-          })}>
-            <Link href={navbarItems[1].link}><b>02</b> {navbarItems[1].title}</Link>
-          </li>
-          <li className={classNames(styles.navbarLinks, {
-            [styles.isLinkActive]: navbarItems[2].link === currentPath,
-          })}>
-            <Link href={navbarItems[2].link}><b>03</b> NASA COLLABORATION</Link>
-          </li>
+
+          {navbarItems.map((item) => (
```
Comment (vdaguenet) [inline components/ui/Navbar.js:53]: Good idea to use a `map`!

**Review by vdaguenet — APPROVED** [review]
Nice!

File: app/destination/page.js (hunk 1 above)
Comment (melo4yxa77796) [reply app/destination/page.js]: i did it

File: components/ui/Footer.js (hunk 2 above)
Comment (melo4yxa77796) [reply components/ui/Footer.js]: I removed all {'' ''}

File: components/ui/Navbar.js (hunk 3 above)
Comment (melo4yxa77796) [reply components/ui/Navbar.js:53]: thank you

**Pull request merged**